Library/Output Templates/Text - PC.txt

```diff
@@ -10,6 +10,7 @@ Disadvantages:@DISADVANTAGES_LOOP_START @DESCRIPTION_PRIMARY [@POINTS];@DISADVAN
 Quirks:@QUIRKS_LOOP_START @DESCRIPTION_PRIMARY [@POINTS];@QUIRKS_LOOP_END
 Skills:@SKILLS_LOOP_START @DESCRIPTION_PRIMARY @RSL [@POINTS]-@SL;@SKILLS_LOOP_END
 Equipment:@EQUIPMENT_LOOP_START @QTY @DESCRIPTION_PRIMARY [$@COST; @WEIGHT];@EQUIPMENT_LOOP_END
+Spells:@SPELLS_LOOP_START @DESCRIPTION_PRIMARY @RSL [@POINTS] [Class: @CLASS; Cost: @MANA_CAST; Maintain: @MANA_MAINTAIN; Time: @TIME_CAST ; Duration:@DURATION];@SPELLS_LOOP_END
```

Comment:
And one more: @SL should probably be there, for the actual skill level, which is almost always the thing you want. @RSL is rarely needed for spells, but could be useful.

Reply:
done !